src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.tsx

```diff
@@ -139,12 +142,32 @@ const TrackPanelListItem = (props: Props) => {
 
     updateGenomeBrowser(newStatus);
 
-    props.updateTrackStates({
-      genomeId: activeGenomeId as string,
-      categoryName,
-      trackId: track.track_id,
-      status: newStatus
-    });
+    if (!activeGenomeId || !activeEnsObjectId) {
+      return;
+    }
+    if (track.track_id.indexOf('track:gene') === 0) {
```

Comment:
`startsWith`?

Does this also need a FIXME comment regarding temporary hack?

Reply:
That's done